docs/architecture/cloud-native/index.md

```diff
@@ -7,7 +7,7 @@ ms.date: 03/07/2019
 
 # Architecting Cloud Native .NET Applications for Azure
 
-![cover image](./media/cover.png)
+![cover image](./media/cover-preview.png)
```

Comment:
instead of creating a new cover, shouldn't you just replace the temp file you have in the directory now?

Reply:
I agree. It's easier to replace the image when the books go out of Preview.